doc/library-contributor.rst

```diff
@@ -10,6 +10,18 @@ Neptune-versioned libraries should read
 
 .. contents::
 
+Submitting Pull Requests
+------------------------
+
+When making a PR to a Neptune-versioned library, the choice of which branch to
+submit a PR against is important. A branch should be chosen as follows:
+* If your change is a bug fix, the PR should be submitted against the oldest
+  supported minor branch.
```

Comment:
This is formatted wrong: https://github.com/gavin-norman-sociomantic/neptune/blob/5ee42034c21ec31a25e6f289451c56ac4fc74a80/doc/library-contributor.rst#id1

Reply:
Bloody rst.